metafacture-io/src/test/java/org/metafacture/io/ObjectFileWriterTest.java

```diff
@@ -19,6 +19,11 @@
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assume.assumeFalse;
+import org.junit.Before;
```

Comment:
Missing empty line before import block.

Reply:
Thx. Addressed in d3a47eeb70166d1ffe66ffd1cf7f3926dbc89c4d.